src/folding/protogalaxy/utils.rs

```diff
@@ -0,0 +1,28 @@
+use ark_ff::PrimeField;
+
+// returns (b, b^2, b^4, ..., b^{2^{t-1}})
+pub fn powers_of_beta<F: PrimeField>(b: F, t: usize) -> Vec<F> {
+    let mut r = vec![F::zero(); t];
+    r[0] = b;
+    for i in 1..t {
+        r[i] = r[i - 1].square();
+    }
+    r
+}
```

Comment:
Agree to change the name, but `powers_of_two` would make it look like it computes the powers of two, but here we compute the powers of the powers of two. I thought about `powers_of_powers_of_two` but it is too confusing, and since @davidnevadoc is sitting on my side, he proposed to use `exponential_powers`, so I've just updated it to `exponential_powers`, and the already existing comment shows a more descriptive description.